SampleUserControlLibrary/SubscriptionKeyPage.xaml.cs

```diff
@@ -158,11 +231,13 @@ private void SaveKey_Click(object sender, RoutedEventArgs e)
             try
             {
                 SaveSubscriptionKeyToIsolatedStorage(SubscriptionKey);
-                MessageBox.Show("Subscription key is saved in your disk.\nYou do not need to paste the key next time.", "Subscription Key");
+                SaveSubscriptionEndPointToIsolatedStorage(SubscriptionEndPoint);
+                //MessageBox.Show(SubscriptionKey + '\n' + SubscriptionEndPoint);
+                MessageBox.Show("Subscription key and endpoint is saved in your disk.\nYou do not need to paste the key next time.", "Subscription Key");
```

Comment:
The previous title of the messagebox `Subscription Key`Please change to `Settings`.